callisto/notification/models.py

```diff
@@ -13,7 +13,7 @@
 @six.python_2_unicode_compatible
 class EmailNotification(models.Model):
     """Record of Email constructed in and sent via the project"""
-    name = models.CharField(blank=False, max_length=50, primary_key=True)
```

Comment:
tangentially, really its because django doesn't automatically fill in defaults when an AutoField is added to a model